.github/workflows/linux.yml

```diff
@@ -1,14 +1,6 @@
 name: Linux CI
 
-on:
```

Comment:
Keep it as is to avoid running the CI for every commit (it is not completely free to run the CI). Or if you prefer, 
```
  push:
    branches:
      - master
  pull_request:
```